port/zos390/omrcel4ro31.c

```diff
@@ -120,7 +122,9 @@ omr_cel4ro31_init(uint32_t flags, const char *moduleName, const char *functionNa
 	if (OMR_ARE_ANY_BITS_SET(flags, OMR_CEL4RO31_FLAG_QUERY_TARGET_FUNC)) {
 		functionBlock->length = functionNameLength;
 		strncpy(functionBlock->functionName, functionName, functionNameLength);
+#if !defined(OMR_EBCDIC)
 		__a2e_l((char *)functionBlock->functionName, functionNameLength);
+#endif /* defined(OMR_EBCDIC) */
```

Comment:
ref: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#header-files
```suggestion
#endif /* !defined(OMR_EBCDIC) */
```

Reply:
Fixed in 0b048ba.  Thanks!